xcode/Subconscious/Shared/Components/BacklinksView.swift

```diff
@@ -9,8 +9,7 @@ import SwiftUI
 
 struct BacklinksView: View {
     var backlinks: [EntryStub]
-    var onRequestDetail: (EntryLink) -> Void
-    var onLink: (_ context: Peer, SubSlashlinkLink) -> Void
```

Comment:
This PR consolidates callbacks and actions for editor links, transcludes, and transclude links. Everything now goes through one signature: `EntryLink`. This struct includes a link and title (which may be used as fallback content in event of creating a new note).